client/src/pages/dashboard/index.tsx

```diff
@@ -16,6 +16,7 @@ import { DataGrid } from "@mui/x-data-grid";
 import BreakdownChart from "components/BreakdownChart";
 import FlexBetween from "components/FlexBetween";
 import Header from "components/Header";
+import MotionBox from "components/motion/MotionBox";
```

Comment:
I accidentally committed the two previous commits right to the main 💀
And I thought there was some protection on it. And there is though.
Anyway, there should have been three commits total on this PR. Let's pretend that there actually is.